specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/examples/WorkspaceCreate.json

```diff
@@ -35,15 +35,32 @@
         "id": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Databricks/workspaces/myWorkspace",
         "type": "Microsoft.Databricks/workspaces",
         "location": "East US 2",
-        "tags": null,
+        "tags": {
+          "environment": "dev"
+        },
         "sku": {
           "name": "skuName"
         },
         "properties": {
           "managedResourceGroupId": "/subscriptions/subid/resourceGroups/myManagedRG",
           "parameters": null,
-          "provisioningState": "Created",
-          "uiDefinitionUri": "https://path/to/workspaceCreateUiDefinition.json"
+          "provisioningState": "Accepted",
+          "authorizations": [
```

Comment:
Are these response for create request or create response

Reply:
this is the response for Create request